runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/graph/GreedyPipelineFuser.java

```diff
@@ -189,7 +189,6 @@ private DescendantConsumers getRootConsumers(PTransformNode rootNode) {
             + "but transform %s executes in environment %s",
         GreedyPipelineFuser.class.getSimpleName(),
         PTransformTranslation.IMPULSE_TRANSFORM_URN,
-        PTransformTranslation.READ_TRANSFORM_URN,
```

Comment:
What's this change for?

Reply:
It was some stuff left over in my workspace which I have gotten rid of now.